hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -595,11 +630,11 @@ void Vgen::emit(const unwind& i) {
 ///////////////////////////////////////////////////////////////////////////////
 
 void Vgen::emit(const andbi& i) {
-  a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
+  a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), updateFlags(i.fl));
```

Comment:
I don't see the value in this discrepancy.  Why not just name the function `UF()`?